charts/kube-otel-stack/values.yaml

```diff
@@ -2,91 +2,178 @@ fullnameOverride: ""
 namespaceOverride: ""
 nameOverride: ""
 
-## Collectors to be installed to the cluster
-##
-collectors:
-  - name: statefulset
-    image: otel/opentelemetry-collector-contrib:0.62.0
+## Auto-Instrumentation resource to be installed in the cluster
+## Can be used by setting the following:
+##  Java: instrumentation.opentelemetry.io/inject-java: "true"
+##  NodeJS: instrumentation.opentelemetry.io/inject-nodejs: "true"
+##  Python: instrumentation.opentelemetry.io/inject-python: "true"
+##  DotNet: instrumentation.opentelemetry.io/inject-dotnet: "true"
+##  OpenTelemetry SDK environment variables only: instrumentation.opentelemetry.io/inject-sdk: "true"
+autoinstrumentation:
+  enabled: false
+  ## The collector name to send traces to
+  collectorTarget: traces
+  propagators:
+    - tracecontext
+    - baggage
+    - b3
+
+  ## Sampler defines the OTEL sampler behavior to be used. Example:
+  ##
+  ## sampler:
+  ##   type: parentbased_traceidratio
+  ##   argument: "0.25"
+  ##
+  sampler:
+    ## The value can be for instance parentbased_always_on, parentbased_always_off, parentbased_traceidratio...
+    type: parentbased_traceidratio
+    ## The value depends on the sampler type.
+    ## For instance for parentbased_traceidratio sampler type it is a number in range [0..1] e.g. 0.25.
+    argument: "0.25"
+
+  ## A list of corev1.EnvVars
+  env: []
+
+  ## https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/overview.md#resources
+  resource: {}
+
+## Custom collectors to be installed to the cluster
+## Matches the structure of tracesCollector and metricsCollector
+collectors: []
+
+## Default collector for tracing
+tracesCollector:
+  enabled: false
+  name: traces
+  image: otel/opentelemetry-collector-contrib:0.64.0
+  mode: deployment
+  replicas: 1
+  resources:
+    limits:
+      cpu: 750m
+      memory: 1000Mi
+    requests:
+      cpu: 750m
+      memory: 1000Mi
+  env:
+    - name: LS_TOKEN
+      valueFrom:
+        secretKeyRef:
+          key: LS_TOKEN
+          name: otel-collector-secret
+  config:
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: "0.0.0.0:4317"
+    processors:
+      memory_limiter:
```

Comment:
optional but nice: add the k8s cluster name as a resource attribute for spans that go through this collector